mmdet/apis/ote/apis/detection/config_utils.py

```diff
@@ -276,3 +284,36 @@ def remove_from_config(config, key: str):
             del config[key]
         else:
             raise ValueError(f'Unknown config type {type(config)}')
+
+def cluster_anchors(config: Config, dataset: DatasetEntity, model):
```

Comment:
And `model` is `torch.nn.Module`, `BaseDetector` or what?

Reply:
Thank you